cards-examples/Web/Web-card-article.md

```diff
@@ -10,229 +10,249 @@ path: /
+</div>
+```
 
+Be sure to include the `mdc-card__action` class on every action for correct positioning. In addition, _button_ icons
+should use the `mdc-card__action--button` class, and _icon_ actions should use the `mdc-card__action--icon` class.
 
-## Theming Cards
+To have a single action button take up the entire width of the action row, use the `--full-bleed` modifier on the row:
 
-Cards support [Material Theming](https://material.io/components/cards/#theming) and can be customized in terms of color, typography and shape.
+```html
+<div class="mdc-card__actions mdc-card__actions--full-bleed">
+  <a class="mdc-button mdc-card__action mdc-card__action--button" href="#">
+    <div class="mdc-button__ripple"></div>
+    <span class="mdc-button__label">All Business Headlines</span>
+    <i class="material-icons mdc-button__icon" aria-hidden="true">arrow_forward</i>
+  </a>
+</div>
+```
 
-### Card theming example
+To display buttons _and_ icons in the same row, wrap them in `mdc-card__action-buttons` and `mdc-card__action-icons`
+elements:
+
+```html
+<div class="mdc-card__actions">
+  <div class="mdc-card__action-buttons">
+    <button class="mdc-button mdc-card__action mdc-card__action--button">
+      <div class="mdc-button__ripple"></div>
+      <span class="mdc-button__label">Read</span>
+    </button>
+    <button class="mdc-button mdc-card__action mdc-card__action--button">
+      <div class="mdc-button__ripple"></div>
+      <span class="mdc-button__label">Bookmark</span>
+    </button>
+  </div>
+  <div class="mdc-card__action-icons">
+   <button class="material-icons mdc-icon-button mdc-card__action mdc-card__action--icon" title="Share">share</button>
+    <button class="material-icons mdc-icon-button mdc-card__action mdc-card__action--icon" title="More options">more_vert</button>
+  </div>
+</div>
+```
 
-API and source code
+##### Icons
 
-* `\<card classes\>`
-  * [Class definition](https://)
-  * [GitHub source](https://github.com/material-components/)
+We recommend using [Material Icons](https://material.io/tools/icons/) from Google Fonts:
 
-_Use the [Shrine theme](https://material.io/design/material-studies/shrine.html) for this example_
+```html
+<head>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+</head>
 ```
-* Display an outlined card 
-* Display a card title "Card title"
-* Display a secondary title "Secondary text"
-* Display text reading "Greyhound divisively hello coldly wonderfully marginally far upon excluding."
-* Display two actions, "Action 1" and "Action 2" with two text buttons
-* Display the sample image ![sample card image of yellow and red tulips](assets/card-sample-image.jpg)
-* Make the card checkable with a "favorites" icon
 
+However, you can also use SVG, [Font Awesome](https://fontawesome.com/), or any other icon library you wish.
+
+#### Combined Example
```

Comment:
Use sentence case for titles: capitalize the first word, and only capitalize following proper nouns.